test/server.cpp

```diff
@@ -131,6 +131,13 @@ TEST_F(ServerTest, 200)
     EXPECT_EQ(200, zfs1_->GET(res.url)->status) << "res.url: " << res.url;
 }
 
+TEST_F(ServerTest, 200_IdNameMapper)
+{
+  resetServer(ZimFileServer::NO_NAME_MAPPER);
+  EXPECT_EQ(200, zfs1_->GET("/ROOT/content/6f1d19d0-633f-087b-fb55-7ac324ff9baf/A/index")->status);
+  EXPECT_EQ(404, zfs1_->GET("/ROOT/content/zimfile/A/index")->status);
```

Comment:
What do you think of having the 2 `EXPECT_EQ()` lines repeated before the `resetServer()` with their expected status codes swapped?

Reply:
Done